packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx

```diff
@@ -1,6 +1,11 @@
 import { Button, Icon, Text } from 'blockchain-info-components'
+import moment from 'moment'
+import React from 'react'
+
 import { BuyOrSell, displayFiat, getOrderDestination } from '../model'
+import { capitalize } from '../../../../utils/capitalize'
```

Comment:
we have alias import for this. so just 
`import { capitalize } from 'utils/capitalize'`